InnerEye/ML/reports/classification_report.ipynb

```diff
@@ -226,9 +245,12 @@
    "source": [
     "if val_metrics_csv.is_file() and test_metrics_csv.is_file() and dataset_csv_path.is_file() and \\\n",
     " dataset_subject_column and dataset_file_column:\n",
-    "    plot_k_best_and_worst_performing(val_metrics_csv=val_metrics_csv, test_metrics_csv=test_metrics_csv, \n",
+    "    for hue in config.class_names:\n",
```

Comment:
This means that even for binary classification we get a header "Class=Default", which looks out of place.

Reply:
Should we force users to give classes always a name?